ocamlformat.opam

```diff
@@ -20,7 +20,7 @@ license: "MIT"
 build: [
   ["ocaml" "tools/gen_version.mlt" "lib/Version.ml" version] {pinned}
   ["dune" "build" "-p" name "-j" jobs]
-  ["dune" "runtest" "-p" name] {with-test & ocaml:version >= "4.08"}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
This change has been made in opam-repository.
Suggested by @kit-ty-kate (https://github.com/ocaml/opam-repository/pull/15753#discussion_r372511317)